ecospheres_migrator/batch.py

```diff
@@ -24,6 +24,25 @@ class FailureTransformBatchRecord(TransformBatchRecord):
     error: str
 
 
+class SkipReasonMessage(Enum):
+    """
+    We don't use `SkipReason.value` for this because we pickle the reason
+    and want to be able to change the associated message inbetween jobs.
+    """
+
+    NO_CHANGES = "Pas de modification lors de la transformation."
+
+
+class SkipReason(Enum):
+    NO_CHANGES = "no changes"
```

Comment:
If string contents have really no use (i.e. always check reason.name and not its value), you can use an `IntEnum` here.

Reply:
I'll do that in the next PR, where I touch this code.